java/browse-wfs-layers/README.md

```diff
@@ -17,32 +17,25 @@ Some WFS services return coordinates in X,Y order, while others return coordinat
 ## How it works
 
 1. Create a `WfsService` object with a URL to a WFS feature service.
-1. Obtain a list of `WfsLayerInfo` from `WfsService.ServiceInfo`.
-1. When a layer is selected, create a `WfsFeatureTable` from the `WfsLayerInfo`.
+2. Obtain a list of `WfsLayerInfo` from `WfsService.ServiceInfo`.
```

Comment:
Some of these readmes have been written from SDKs using .NET/Qt and so have slightly different method calls. For our SDK, we should use the following:
```suggestion
2. Obtain a list of `WfsLayerInfo` from `WfsService.getServiceInfo`.
```